src/components/notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+
+interface Props {
+  setError: (errVal: string | boolean) => void;
+  errorText: string | true;
+}
+
+export const Notification: React.FC<Props> = ({ errorText, setError }) => {
+  useEffect(() => {
```

Comment:
Add 'setError' to the dependency array of useEffect to avoid stale state issues.